src/browser/modules/D3Visualization/lib/visualization/renders/init.js

```diff
@@ -134,6 +135,75 @@ const nodeRing = new Renderer({
   onTick: noop
 })
 
+const nodeImage = new Renderer({
+  onGraphChange (selection, viz) {
+    const pattern = selection.selectAll('pattern').data(function (node) {
+      if (node.propertyMap.image_url) {
+        return [node]
+      } else {
+        return []
+      }
+    })
+
+    pattern
+      .enter()
+      .append('pattern')
+      .attr({
+        id (id) {
+          return 'img-fill-' + id.id
+        },
+        patternUnits: 'userSpaceOnUse',
+        x: -39,
+        y: -39,
+        height: 156,
+        width: 156
```

Comment:
`x`, `y`, `height`, `width` shouldn't be hard-coded here.  Calculate based on node radius instead, or it won't look right when the user configures different sizes of nodes in their GraSS.  Ran into this myself 😛 ... see [here](https://github.com/neo4j/neo4j-browser/pull/887/files#diff-da817c60492d350eb701f5f1c3e687c1R147)